Formula/packetq.rb

```diff
@@ -0,0 +1,26 @@
+class Packetq < Formula
+  desc "SQL-frontend to PCAP-files"
+  homepage "https://www.dns-oarc.net/tools/packetq"
+  url "https://github.com/DNS-OARC/PacketQ/archive/v1.3.1.tar.gz"
+  sha256 "7c649207cc35a19fa005f09ba368466606b5d4c20715d5ef0decb21f329b568c"
+
+  head do
+    url "https://github.com/DNS-OARC/PacketQ.git"
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  def install
+    system "./autogen.sh"
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "make", "test"
```

Comment:
`make test` is not something we run except for security software, nor it could it be run in the test block because the build environment is gone at this point. This will need to be a test that verifies the functionality of the package after it has already been installed.